crates/swc_css_minifier/src/compress/urange.rs

```diff
@@ -0,0 +1,100 @@
+use swc_css_ast::*;
+use swc_css_visit::{VisitMut, VisitMutWith};
+
+pub fn compress_urange() -> impl VisitMut {
+    CompressUrange {}
+}
+
+struct CompressUrange {}
+
+impl CompressUrange {
+    fn remove_leading_zeros(&mut self, value: &str) -> String {
+        let mut result = String::new();
+        let mut is_leading = true;
+
+        for char in value.chars() {
```

Comment:
Please use different name than `char`, which is a builtin type.
This distracts review process.

Reply:
This still applies